habitat-lab/habitat/tasks/rearrange/sub_tasks/nav_to_obj_task.py

```diff
@@ -135,18 +140,116 @@ def reset(self, episode: Episode):
 
         # in the case of Stretch, force the agent to look down and retract arm with the gripper pointing downwards
         if isinstance(sim.robot, StretchRobot):
+            camera_rot = (
+                -1.7375 if self._pick_init or self._place_init else 0.0
+            )
             sim.robot.arm_motor_pos = np.array(
-                [0.0] * 4 + [0.775, 0.0, -1.57000005, 0.0, 0.0, -0.7125]
```

Comment:
again i really dislike having all these hard coded values - espeically if they are intended to be same as real world. we should put them elsewhere

Reply:
Since, these are used for as initial poses at the time of training skills this can't be pulled from home-robot. I moved these to a separate file in habitat-lab.